py/torch_tensorrt/dynamo/conversion/impl/elementwise/base.py

```diff
@@ -103,9 +104,13 @@ def convert_binary_elementwise(
     # dtype but we don't have a way to detect whether it makes sense for the
     # scalar to be float or half. Hence we go with the lhs dtype.
     if is_lhs_trt_tensor and isinstance(rhs_val, (float, int)):
-        rhs_val = torch.tensor([rhs_val], dtype=lhs_dtype)
```

Comment:
Why is this getting converted to `np` array in place of the `torch.Tensor`. Is that because the `torch.Tensor` was leading to clones getting produced in the graph?

Reply:
Creating `torch.Tensor` objects within the `torch.compile` scope causes them to be "Fake-ified" which removes the data contained within them.